src/Backend/SgateShopgatePlugin/Components/Favorites.php

```diff
@@ -0,0 +1,388 @@
+<?php
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+use Shopgate\Helpers\WebCheckout;
+
+class Shopware_Plugins_Backend_SgateShopgatePlugin_Components_Favorites
```

Comment:
can you use PSR-2 class names here instead of PSR-0? e.g. `Shopware\Plugins\Backend\SgateShopgatePlugin\Components\Cart;`
We are doing it for the Helper, I was hoping that we could also do it for these components as well. It will be cleaner in the long run.